app/pulp/app/serializers/repository.py

```diff
@@ -63,14 +63,88 @@ class ImporterSerializer(MasterModelSerializer):
     """
     Every importer defined by a plugin should have an Importer serializer that inherits from this
     class. Please import from `pulp.app.serializers` rather than from this module directly.
-
-    Every subclass must override the `_href` field with a `RepositoryNestedIdentityField` that
-    defines the view_name.
     """
+    # _href is normally provided by the base class, but Importers's
+    # "name" lookup field means _href must be explicitly declared.
+    _href = serializers.HyperlinkedIdentityField(
+        view_name='importers-detail',
+        lookup_field='name',
+    )
+
     name = serializers.CharField(
         help_text='A name for this importer, unique within the associated repository.'
     )
+    last_updated = serializers.DateTimeField(
+        help_text='Timestamp of the most recent update of this configuration.',
+        read_only=True
+    )
+
+    feed_url = serializers.CharField(
+        help_text='The URL of an external content source.',
+        required=False,
+    )
+
+    validate = serializers.BooleanField(help_text='Whether to validate imported content.')
```

Comment:
Consider adding required=False. This field has default value in the model but that does not implies required=False, I've tested it. It is only the case when default value is specified in serializer.